Formula/lib/libadwaita.rb

```diff
@@ -25,12 +25,14 @@ class Libadwaita < Formula
     sha256 x86_64_linux:   "9c9ed668d719bab47cc21c920d3214de8c81b9447232ba8f0bce968053a41b05"
   end
 
+  depends_on "cmake" => :build
   depends_on "gettext" => :build
   depends_on "gobject-introspection" => :build
   depends_on "meson" => :build
   depends_on "ninja" => :build
   depends_on "pkg-config" => [:build, :test]
   depends_on "vala" => :build
+  depends_on "appstream"
   depends_on "gtk4"
 
```

Comment:
```suggestion

  uses_from_macos "python" => :build
```

Reply:
cc @danyeaw

Reply:
Hey @chenrui333, does `uses_from_macos` work for Linux failures?

Reply:
yes

Reply:
here is PR ref, https://github.com/Homebrew/homebrew-core/pulls?q=is%3Apr+uses_from_macos+%22python%22

Reply:
Thanks @chenrui333, I pushed a hopefully final update with this change. I just got confused with the term uses_from_macos, but I understand it is more about using the system Python.